addons/survey/models/survey_user_input.py

```diff
@@ -784,20 +784,14 @@ def _get_answer_score_values(self, vals, compute_speed_score=True):
 
         if compute_speed_score and answer_score > 0:
             user_input = self.env['survey.user_input'].browse(user_input_id)
-            session_speed_rating = user_input.exists() and user_input.is_session_answer and user_input.survey_id.session_speed_rating
```

Comment:
We do not really need to have a session for the time to work, and we can technically change the naming of session_speed_rating and have this feature also for normal scored surveys, but I'm not sure if I should still change that logic since it was not in the task description... I'll add the constraint for sessions only and discuss it with Francois once he comes back

Reply:
I would not go in the grey area. Maybe keep it for sessions exclusively then? I see you reintroduced the condition, seems ok :)